tests/integration_test.py

```diff
@@ -64,5 +64,5 @@ def test_powerchainger_smoke():
 
     df = pd.DataFrame(data, index=ds_parser.index, columns=["inference"])
 
-    expected = np.load(expected_file)
+    expected = np.load(expected_file, allow_pickle=True)
```

Comment:
Out of curiosity: why include `allow_pickle=True` here?

Reply:
Huh. During development I needed it so `numpy` does not throw an error. I wanted to look up which one exactly, but now I can't reproduce it anymore. Weird. I'll remove it.

Reply:
Some CI tests fail because of errors in the synthetic data. They did pass locally. I'll fix them and update the PR.

Reply:
Nice. Now only the SonarCloud CI fails, but that's covered through https://github.com/ESI-FAR/TeSoPs-ELECTRIcity_NILM/pull/44 .